packages/corvid-fake-local-mode-editor/src/editor.js

```diff
@@ -139,6 +140,10 @@ const loadEditor = async (
     sendRequest(socket, "UPDATE_DOCUMENT", editorState.siteDocument);
 
   const saveCodeFiles = async () => {
+    if (extraArgs.failOnClone === "true") {
```

Comment:
why does "failOnClone" mean we need signal a failure inside the "saveCodeFiles" method ?

Reply:
i want it to fail only after it created some files

Reply:
moved